packages/flutter_test/lib/src/matchers.dart

```diff
@@ -457,6 +459,36 @@ Matcher matchesSemanticsData({
   );
 }
 
+/// Asserts that the currently rendered widget meets the provided accessibility
+/// `guideline`.
+///
+/// This matcher requires the result to be awaited and for semantics to be
+/// enabled first.
+///
+/// ## Sample code
+///
+/// ```dart
+/// final SemanticsHandle handle = tester.ensureSemantics();
+/// await meetsGuideline(tester, meetsGuideline(textContrastGuideline));
```

Comment:
include disposing the handle in this example?

Reply:
Done